content/docs/user-guide/data-management/remote-storage/ssh.md

```diff
@@ -1,117 +1,117 @@
+[ssh server]: https://www.ssh.com/academy/ssh/server
+[SSH URL]: https://tools.ietf.org/id/draft-salowey-secsh-uri-00.html#sshsyntax
 
-- `url` - remote location, in a regular
-  [SSH format](https://tools.ietf.org/id/draft-salowey-secsh-uri-00.html#sshsyntax).
-  Note that this can already include the `user` parameter, embedded into the
-  URL:
+<admon type="warn">
 
-  ```cli
-  $ dvc remote modify myremote url \
-                      ssh://user@example.com:1234/path
-  ```
+DVC requires both SSH and SFTP access to work with SSH remote storage. Check
+that you can connect both ways with tools like [`ssh`] and `sftp` (GNU/Linux).  
+Note that your server's SFTP root might differ from its physical root (`/`).
 
-  ⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.
-  Please check that you are able to connect both ways with tools like `ssh` and
-  `sftp` (GNU/Linux).
+[`ssh`]: https://www.ssh.com/academy/ssh/command
 
-  > Note that your server's SFTP root might differ from its physical root (`/`).
+</admon>
 
-- `user` - user name to access the remote:
+By default, authentication credentials (user name, password or private key,
+etc.) not found in the URL are loaded from [SSH configuration]. You can also set
+them directly with DVC.
 
-  ```cli
-  $ dvc remote modify --local myremote user myuser
-  ```
+[ssh configuration]: https://www.ssh.com/academy/ssh/config
 
-  The order in which DVC picks the user name:
+## Custom authentication
 
-  1. `user` parameter set with this command (found in `.dvc/config`);
-  2. User defined in the URL (e.g. `ssh://user@example.com/path`);
-  3. User defined in the SSH config file (e.g. `~/.ssh/config`) for this host
-     (URL);
-  4. Current system user
+2 parameters that are commonly included in an SSH URL are user name and
+sometimes port. These can be set (or overridden) as follows:
 
-- `port` - port to access the remote.
+```cli
+$ dvc remote modify myremote user myuser
+$ dvc remote modify myremote port 2222
+```
 
-  ```cli
-  $ dvc remote modify myremote port 2222
-  ```
+Order in which DVC picks these values when defined in multiple places:
 
-  The order in which DVC decide the port number:
+1. Value set in these `user`/`port` params (DVC-specific config)
+2. User/port embedded in the `url`, if any (e.g. `ssh://user@example.com:2222`)
+3. `User`/`Port` defined for the host in SSH config
+4. Default values: Current system user; Standard SSH port 22
 
-  1. `port` parameter set with this command (found in `.dvc/config`);
-  2. Port defined in the URL (e.g. `ssh://example.com:1234/path`);
-  3. Port defined in the SSH config file (e.g. `~/.ssh/config`) for this host
-     (URL);
-  4. Default SSH port 22
+<admon type="warn">
 
-- `keyfile` - path to private key to access the remote.
+The `dvc remote modify --local` flag is needed to write sensitive user info to a
+Git-ignored config file (`.dvc/config.local`) so that no secrets are leaked
+through Git. See `dvc config`.
```

Comment:
> See `dvc config`.

Should replace for config guide when #4379 is merged (here and in other guides where it's already mentioned).